WindowsServerDocs/manage/windows-admin-center/azure/index.md

```diff
@@ -83,7 +83,7 @@ For more info, see [Deploy a Cloud Witness for a Failover Cluster](../../../fail
 - **Connect your on-premises servers to an Azure Virtual Network with [Azure Network Adapter](https://aka.ms/WACNetworkAdapter)**
 Let Windows Admin Center simplify setting up a point-to-site VPN from an on-premises server into an Azure virtual network.
 
-- **Make Azure VMs look like your on-premises network with [Azure Extended Network](https://go.microsoft.com/fwlink/?linkid=2109517&clcid=0x409)**
```

Comment:
- [ ] 

- [ ]

Reply:
If you have got anything relevant to contribute, please be more clear in your feedback.